healpy/pixelfunc.py

```diff
@@ -942,13 +936,7 @@ def nside2order(nside):
 
     >>> np.all([hp.nside2order(2**o) == o for o in range(30)])
     True
-
-    >>> hp.nside2order(7)
-    Traceback (most recent call last):
```

Comment:
This should still raise an exception because the concept of an order is tied to nside being a power of 2. Please bring back this doctest.